episodes/144/README.md

```diff
@@ -0,0 +1,256 @@
+# Episode 144 : Exploring The State of K8s on Windows
+
+- Hosted by @jayunit100
+- Recording date: 2021-01-28 
+
+<!--- Thumbnailed embed of the video, n8Xo_ghCIOSY is the video id from the youtube url --->
+
+<a href="https://www.youtube.com/watch?v=WnXnO97tsNo" target="_blank"><img src="https://i.ytimg.com/vi/WnXnO97tsNo/maxresdefault.jpg" border="10" /></a>
+
+## Table of Contents
+
+- 00:00:00 - Welcome to TGIK!
+- 00:00:00 - Week in Review
```

Comment:
I generally go through and find the main things in the timeline. At least call out when you start talking about the main topic. If you copy/paste this table into the youtube description then it creates chapter marks for folks.

Reply:
ah yeah ! will do that
perfect